02-localproject.Rmd

```diff
@@ -343,21 +345,98 @@ As mentioned earlier, a .gitignore file was created with our R Project. You can
+
+Whilst you've got the Global options open, another useful option to set is the default terminal to use in RStudio. The default is the Windows Command Prompt, but [Git BASH](#opening-git-bash) is generally easier to use. To update this, keep Global options open and select the `Terminal` options panel just below Git/SVN.
+
+Here you'll see an option labeled `New terminals open with:`. If it isn't already set to `Git Bash`, we recommend changing it to that, even if you're primarily planning on using the RStudio git GUI.
+
+### Cloning an existing project
+
+The easiest way to get set up with a repository in RStudio is to use the `New Project...` option under `File` in the menu bar. When you select `New Project...`, you'll then see three options: `New directory`, `Existing directory` and `Version control`. Assuming you've already got your repository set up on Dev Ops or GitHub, select `Version control` and then `Git` on the next panel that comes up. Once you've done that, you'll see the options below.
+
+![RStudio Clone Git Repository](www/images/chapter2_RStudio_Clone.PNG)
+
+Here you can enter your existing `Repository URL` and choose a root directory for where to keep it locally. The `Project directory name` option will be auto-filled once you provide a `Repository URL`. For the root directory where the repository directory will be cloned, we recommend not using your standard One Drive folder system as this can cause issues if both One Drive and git are trying to manage file changes. Instead it's best to navigate either to `C:\` or `C:\Users\<username>\` and create a suitable root folder (e.g. `my_repos`) to store your repositories in (and pinning this folder to Quick access in File Explorer can help with finding your repository directories and files quickly in Windows).
+
+Once you've entered your repository URL and chosen a directory in which to clone your repository, click `Create Project` and RStudio will do the rest.
+
+### Overview of the RStudio git GUIs
+
+There are three key areas within RStudio for accessing git functionality. The the Git Pane, the terminal and the RStudio menu bar.
+
+#### Terminal ([Git BASH](#opening-git-bash))
+
+The terminal offers all the [Git BASH](#opening-git-bash) command line functionality already described. It can be accessed via the terminal pane usually found next to the R console pane.
+
+#### Git pane
+
+The Git pane offers an excellent quick look at the status of your repository, whilst offering quick links for viewing or committing changes, pulling and pushing, viewing the history, accessing the project settings, and creating and switching branches. Effectively all the common things you need to do with `Git`. The image below shows the git pane opened for a local clone of the repo for this guidance. It conssists of it's own menu bar with buttons for each of the options listed above, alongside a preview pane showing any files that have been modified, added or removed since your last commit.
```

Comment:
Might be worth talking a bit about the "status" - what do the different letters mean?